testbed/testbed/trace_exporters.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testbed
+
+import (
+	"fmt"
+
+	"contrib.go.opencensus.io/exporter/jaeger"
```

Comment:
Probably not worth doing now but this can be moved to collector exporter (or the OTel go one), likely only worth doing after moving the internal format to OTel.

Reply:
Yeah, this is an existing code that works and is tested. I don't want to touch it for now.